core/src/test/java/org/testcontainers/containers/wait/internal/InternalCommandPortListeningCheckTest.java

```diff
@@ -0,0 +1,44 @@
+package org.testcontainers.containers.wait.internal;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+
+import static com.google.common.primitives.Ints.asList;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertThrows;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+
+public class InternalCommandPortListeningCheckTest {
+
+    private GenericContainer nginx;
+
+    @Before
+    public void setUp() {
+        nginx = new GenericContainer<>("nginx:1.9.4");
```

Comment:
why not as a rule? :)

Reply:
You're right, I thought this avoided a problem but it doesn't!